contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -195,9 +195,10 @@ public File fetch(String sourceFileUrl,
                       String storeName,
                       long pushVersion,
                       MetadataStore metadataStore) throws Exception {
+        // FIXME: Stop using an admin client to talk to yourself, silly Voldemort!
```

Comment:
Well, as Voldemort split his soul into horcruxes. It's possible that he likes to talk with himself :)